roles/sshd/templates/sshd_config

```diff
@@ -1,4 +1,8 @@
 Port 22
+{% if inventory_hostname in groups['jumphost'] %}
+Port 80
```

Comment:
I don't like running ssh on a non standard port. It is again one more thing you have to remember when to access the host in a pinch.
And if you would do it, port 80 makes less sense than 443 as port 80 can still easily be restricted to web traffic as it should  not be encrypted.
But again: please use a vpn when accessing from some remote site (eg: a ski resort)

Reply:
Why the extra need for a VPN that a lot of users don't have? The data is already encrypted, so a VPN only complicates the setup and adds latency. Using port 443 instead of 80 would be better.  I can change that, but I'm wondering if the additional port for SSH  be it 80 or 443 is not conflicting with the stuff from the ```prom_proxy``` role. Just looked at the ```nginx``` config for that role, but I'm not so familiar with the ```nginx``` configs...

Reply:
Note that it is an extra port: we continue to use SSH on the default port 22. The extra port is only for those behind silly firewalls.